### PR TITLE
plugins.bfmtv : Update regular expression

### PR DESCRIPTION
**Pull request by BZHDeveloper**
Update regular expression with https scheme. BFM urls are secure now

### CODE REVIEW

**Comment by beardypig** [comment]
Thanks @BZHDeveloper. Looks like you'll need to update the tests though :)

**Comment by BZHDeveloper** [comment]
of course ^^

**Comment by gravyboat** [comment]
Thanks @BZHDeveloper!

File: tests/test_plugin_bfmtv.py [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
+        self.assertTrue(BFMTV.can_handle_url("https://rmcsport.bfmtv.com/mediaplayer/live-bfm-sport/"))
+        self.assertTrue(BFMTV.can_handle_url("https://rmcdecouverte.bfmtv.com/mediaplayer-direct/"))
+        self.assertTrue(BFMTV.can_handle_url("https://www.bfmtv.com/mediaplayer/replay/premiere-edition/"))
+        self.assertTrue(BFMTV.can_handle_url("https://bfmbusiness.bfmtv.com/mediaplayer/replay/good-morning-business/"))
+        self.assertTrue(BFMTV.can_handle_url("https://rmc.bfmtv.com/mediaplayer/replay/les-grandes-gueules/"))
+        self.assertTrue(BFMTV.can_handle_url("https://rmc.bfmtv.com/mediaplayer/replay/after-foot/"))
+        self.assertTrue(BFMTV.can_handle_url("https://www.01net.com/mediaplayer/replay/jtech/"))
+        self.assertTrue(BFMTV.can_handle_url("https://www.bfmtv.com/politique/macron-et-le-pen-talonnes-par-fillon-et-melenchon-a-l-approche-du-premier-tour-1142070.html"))
+        self.assertTrue(BFMTV.can_handle_url("https://rmcdecouverte.bfmtv.com/mediaplayer-replay/?id=6714&title=TOP%20GEAR%20:PASSION%20VINTAGE"))
+
```
Comment (gravyboat) [inline tests/test_plugin_bfmtv.py]: Bah I just realized there's this extra space on line 22, we'll have to clean that up at some point.

Comment (gravyboat) [reply tests/test_plugin_bfmtv.py]: @BZHDeveloper No need to apologize! I should have caught it on the review.

**Pull request merged**